src/DatePicker/CalendarToolbar.js

```diff
@@ -80,32 +82,28 @@ class CalendarToolbar extends React.Component {
     const prevButtonIcon = this.context.muiTheme.isRtl ? <NavigationChevronLeft /> : <NavigationChevronRight />;
 
     return (
-      <Toolbar style={styles.root} noGutter={true}>
+      <div style={styles.root}>
+        <IconButton
```

Comment:
The ripple circle is messed up on these buttons:

![img](https://dl.dropboxusercontent.com/s/yathat1sad6yzbt/2016-04-26%20at%209.08%20AM.png)